CMakeLists.txt

```diff
@@ -70,6 +71,28 @@ string(TOUPPER ${ONEDPL_BACKEND} ONEDPL_BACKEND)
 message(STATUS "Using parallel policies with ${ONEDPL_BACKEND} backend")
 string(TOLOWER ${ONEDPL_BACKEND} ONEDPL_BACKEND)
 
+if (ONEDPL_ENABLE_SIMD)
+    if (NOT CMAKE_CXX_COMPILER MATCHES ".*dpcpp(-cl)?$")
+        foreach(_simd_flag -fopenmp-simd -qopenmp-simd /Qopenmp-simd -openmp:experimental -openmp-simd)
```

Comment:
`-fiopenmp-simd`, `/Qiopenmp-simd` could also be valid options (https://software.intel.com/content/www/us/en/develop/documentation/oneapi-dpcpp-cpp-compiler-dev-guide-and-reference/top/compiler-reference/compiler-options/compiler-option-details/offload-compilation-options-and-openmp-options-and-parallel-processing-options/fiopenmp-qiopenmp.html)

Reply:
Do you want me to add them? Should I add them _before_ current options or _after_? Other options are not considered after the first suitable one is found.

Reply:
I'd put it _before_ current options